go.mod

```diff
@@ -27,6 +27,7 @@ require (
 )
 
 replace (
+	github.com/gardener/gardener-extensions => github.com/gardener/gardener-extensions v1.3.0
```

Comment:
This here is quite strange... How could version `v1.2.1-0.20200129183102-afdd6bad562a` just disappear from the git commits? 
Assuming it comes from `github.com/gardener/gardener v1.0.2`, how does gardener itself compile?
If possible, I would try to solve this on the root of the problem and not here.

Reply:
What is your suggestion to solve this issue? I am not sure why it can not find the corresponding version anymore. It comes with the `github.com/gardener/gardener-extension-provider-azure` dependency ([go.mod](https://github.com/gardener/gardener-extension-provider-azure/blob/v1.3.0/go.mod#L12)). This itself has a dependency to `github.com/gardener/gardener-extension-networking-calico`, which has the broken dependency [here](https://github.com/gardener/gardener-extension-networking-calico/blob/ca90704ba78e1bd09677a43b90dd969992b32504/go.mod#L8). There is no newer version of the azure extension available, so we can't fix it by upgrading the provider dependencies.

Reply:
OK interesting, I thought go mod was complaining about `github.com/gardener/gardener-extensions` itself, not some transitive dependency futher down the tree.

In any case, what I meant is that this provider just imports `github.com/gardener/gardener`. If that package has a broken dependency it should be fixed in `github.com/gardener/gardener`, not here.

Reply:
I removed this change from the PR